From 48a2718140bbdc33e57697001fbcb5d7b9b68101 Mon Sep 17 00:00:00 2001
From: Lok Jian Ming <jianminglok@hotmail.com>
Date: Fri, 18 Mar 2016 10:35:14 +0800
Subject: [PATCH] Update common.mk

---
 config/common.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index b86ffee..2e782bf 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -251,7 +251,7 @@ endif
 
 # Filter out random types, so it'll reset to UNOFFICIAL
 ifeq ($(filter RELEASE NIGHTLY SNAPSHOT EXPERIMENTAL,$(CM_BUILDTYPE)),)
-    CM_BUILDTYPE :=
+    CM_BUILDTYPE := NIGHTLY
 endif
 
 ifdef CM_BUILDTYPE
@@ -276,8 +276,8 @@ ifdef CM_BUILDTYPE
         endif
     endif
 else
-    # If CM_BUILDTYPE is not defined, set to UNOFFICIAL
-    CM_BUILDTYPE := UNOFFICIAL
+    # If CM_BUILDTYPE is not defined, set to NIGHTLY for OTA updates
+    CM_BUILDTYPE := NIGHTLY
     CM_EXTRAVERSION :=
 endif
 
